client/app/styles/rjsf/InlineFieldTemplate.tsx

```diff
@@ -73,15 +75,27 @@ function InlineFieldTemplate({
         }}
       >
         {children}
-        {isErrors && (
-          <div
-            className="text-red-500 absolute right-[-40px] pt-2"
-            role="alert"
-          >
-            <AlertIcon />
-          </div>
-        )}
       </Grid>
+      {isErrors && (
+        <Grid
+          item
+          xs={12}
+          md={4}
+          role="alert"
+          sx={{
+            display: "flex",
+            alignItems: "center",
+            color: "error.main",
+            marginLeft: {
+              xs: "0",
+              md: "16px",
+            },
+          }}
+        >
+          <AlertIcon classNames="hidden md:block mr-4" />
```

Comment:
The icon changes size depending on the length of the error message: 
![image](https://github.com/bcgov/cas-registration/assets/77306764/fd5fe705-10fd-4fbb-b8e6-1cee469a5c0f)

Reply:
Nice catch! Fixed this as well as another breakpoint issue where the icon wasn't hiding correctly in small breakpoint range after the breakpoint changed to vertical layout. Thanks 🙇‍♂️

Reply:
Looks good at all sizes